docker-compose.yml

```diff
@@ -29,6 +29,8 @@ services:
     volumes:
       - .:/var/www/html
       - ./data/data:/data
+      - /var/www/html/mod
```

Comment:
Your right, but I thought this was better that listing out all the Composer installed modules since that list is likely to grow